src/styles/blocks/compare.scss

```diff
@@ -0,0 +1,61 @@
+.bike {
+  &__item {
+    margin-bottom: 40px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @include onTablet {
+      margin-bottom: 48px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.compare {
+  &__photo-container {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    column-gap: 24px;
+
+    margin-bottom: 32px;
+    height: 100%;
+  }
+
+  &__photo {
+    width: 100%;
+    grid-column: 1/5;
+
+    transition: all 1s ease;
```

Comment:
Always add transition property. It's hard for the browser to control all properties

Reply:
fixed.